### ISSUE TITLE
update 'save and run' button in Mocha Specs

### ISSUE THREAD

**Issue by MelMepham** [TradeMe/tractor#246]
BEHAVIOR:
When you press `save and run` it pops up saying `This will overwrite "xxx". Continue?` if you press cancel it still runs the test.
If you press `continue` and are changing the underlying javascript it will over ride the code - making it difficult to figure out how to run the test while keeping changes custom.

ISSUE:
* I want to run a test without saving it easily.
* I assumed if you pressed cancel it would not run the test. - but by pressing cancel it is doing the thing I want it to do.

FIX:
Better wording to understand what the actual behaviour is
OR
Add an extra button to `run without saving` and make the `cancel` behaviour not run the test.

**Comment by phenomnomnominal** [comment]
Because the test runs on the generated JavaScript code, the code *has* to be saved before the test can run. I guess it might be possible to save to a temp location and run it, but that sounds hard!

**Comment by MelMepham** [comment]
The test does run if you say 'cancel'..

**Comment by phenomnomnominal** [comment]
Yeah, cause it saves the file. The “cancel” is definitely broken, but I don’t know exactly how you’d like it to work?
